static/options.interfaces.ts

```diff
@@ -62,4 +62,7 @@ export type Options = {
     sentryEnvironment?: string;
     compileOptions: Record<LanguageKey, string>;
     tools: Record<LanguageKey, Record<string, Tool>>;
+    supportsExecute: boolean;
```

Comment:
There's https://github.com/compiler-explorer/compiler-explorer/issues/4261 that asks for this kind of work, so thanks for starting it :)